extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -107,8 +115,40 @@ function ($, mw, loader, nirvana, labeling) {
 	 */
 	function processInstance(modalInstance) {
 		/* Submit template type edit form on Done button click */
-		modalInstance.bind('done', function runSave() {
+		modalInstance.bind('done', function runSave(e) {
 			processSave(modalInstance);
+
+			// Track - primary-button click
+			tracker.track({
+				trackingMethod: 'both',
+				category: 'template-classification-dialog',
+				action: 'primary-button',
```

Comment:
Are you sure it's not just tracker.ACTIONS.CLICK here? IMO 'primary-button' could be a part of a label.